src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -36,103 +48,94 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
-    const iconType = 'pf';
-    const heading = (
-      <div className="ServiceList-Heading">
-        <div className="ServiceList-Title">
-          <div className="component-label">Service</div>
-          <Link to={this.serviceLink(namespace, serviceName)}>{serviceName}</Link>
-        </div>
-      </div>
+    return (
+      <ListItem key={`AppService_${serviceName}`}>
+        <Link to={this.serviceLink(namespace, serviceName)}>{serviceName}</Link>
+      </ListItem>
     );
-    const content = (
-      <ListViewItem
-        leftContent={<ListViewIcon type={iconType} name={iconName} />}
-        key={`AppService_${serviceName}`}
-        heading={heading}
-      />
-    );
-    return content;
   }
 
   renderEmptyItem(type: string) {
     const message = 'No ' + type + ' found for this app.';
-    return <ListViewItem description={message} />;
+    return <DataListCell> {message} </DataListCell>;
   }
 
   workloadList() {
     const ns = this.props.app.namespace.name;
     const workloads = this.props.app.workloads;
-    return workloads.length > 0
-      ? workloads.map(wkd => this.renderWorkloadItem(ns, wkd))
-      : this.renderEmptyItem('workloads');
+    const workloadList =
+      workloads.length > 0 ? workloads.map(wkd => this.renderWorkloadItem(ns, wkd)) : this.renderEmptyItem('workloads');
+
+    return [
+      <DataListCell key="workload-icon" isIcon={true}>
+        <Badge>W</Badge>
+      </DataListCell>,
+      <DataListCell key="workload-list" className="resourceList">
+        <h3>Workloads</h3>
+        <List>{workloadList}</List>
+      </DataListCell>
+    ];
   }
 
   serviceList() {
     const ns = this.props.app.namespace.name;
     const services = this.props.app.serviceNames;
-    return services.length > 0
-      ? services.map(sn => this.renderServiceItem(ns, this.props.app.name, sn))
-      : this.renderEmptyItem('services');
+    const serviceList =
+      services.length > 0
+        ? services.map(sn => this.renderServiceItem(ns, this.props.app.name, sn))
+        : this.renderEmptyItem('services');
+
+    return [
+      <DataListCell key="service-icon" isIcon={true}>
+        <Badge>S</Badge>
+      </DataListCell>,
+      <DataListCell key="service-list" className="resourceList">
+        <h3>Services</h3>
+        <List>{serviceList}</List>
+      </DataListCell>
+    ];
   }
 
   render() {
     const app = this.props.app;
     return app ? (
-      <div className="card-pf">
-        <div className="card-pf-body">
-          <Row>
-            <Col xs={12} sm={6} md={4} lg={4}>
-              <ListView>{this.workloadList()}</ListView>
-            </Col>
-            <Col xs={12} sm={6} md={4} lg={4}>
-              <ListView>{this.serviceList()}</ListView>
-            </Col>
-            <Col xs={0} sm={0} md={1} lg={1} />
-            <Col xs={12} sm={6} md={3} lg={3}>
-              <div className="progress-description">
+      <Grid gutter="lg">
+        <GridItem span={6}>
```

Comment:
be care with gutter prop is not working I opened an issue about https://github.com/patternfly/patternfly-react/issues/2732.  The workaround is add the style.

There is a PR https://github.com/patternfly/patternfly-react/pull/2421 to change this to hasgutter. the gutter applied is the md one with 1 rem so if you want more space you must to add each GridItem style={{ margin: '0px 10px 0 10px' }}